sdk/turing/generated/model/ensembler1.py

```diff
@@ -0,0 +1,195 @@
+"""
+    Turing Minimal Openapi Spec for SDK
+
+    No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)  # noqa: E501
+
+    The version of the OpenAPI document: 0.0.1
+    Generated by: https://openapi-generator.tech
+"""
+
+
+import re  # noqa: F401
+import sys  # noqa: F401
+
+from turing.generated.model_utils import (  # noqa: F401
+    ApiTypeError,
+    ModelComposed,
+    ModelNormal,
+    ModelSimple,
+    cached_property,
+    change_keys_js_to_python,
+    convert_js_args_to_python_args,
+    date,
+    datetime,
+    file_type,
+    none_type,
+    validate_get_composed_info,
+)
+
+def lazy_import():
+    from turing.generated.model.ensembler_docker_config import EnsemblerDockerConfig
+    from turing.generated.model.ensembler_standard_config import EnsemblerStandardConfig
+    globals()['EnsemblerDockerConfig'] = EnsemblerDockerConfig
+    globals()['EnsemblerStandardConfig'] = EnsemblerStandardConfig
+
+
+class Ensembler1(ModelNormal):
```

Comment:
There are two models named `Ensembler` in the openapi spec now, hence this `Ensembler1` name here.  
I would suggest to rename https://github.com/gojek/turing/blob/1f22856d7f5fd647ef3bbb75fe889ab73e29d1d0/api/api/specs/routers.yaml#L680
into `EnsemblerConfig`, to follow the name of the DB table where these entities are stored.

Reply:
Ok I'll be renaming this class to `RouterEnsemblerConfig` in order to prevent overlapping with the `EnsemblerConfig` for batching jobs.